package.json

```diff
@@ -3,9 +3,17 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@emotion/react": "^11.10.5",
+    "@emotion/styled": "^11.10.5",
+    "@mui/icons-material": "^5.11.0",
+    "@mui/material": "^5.11.7",
```

Comment:
Please be careful to read through the final rubric. Styling section mentions that it is discouraged to use 3rd party CSS libraries.